vendor/bat-native-ads/src/bat/ads/internal/ad_targeting/processors/contextual/text_classification/text_classification_processor.cc

```diff
@@ -31,6 +35,17 @@ std::string GetTopSegmentFromPageProbabilities(
   return iter->first;
 }
 
+chrome_lang_id::NNetLanguageIdentifier::Result detect_page_language(const std::string& text) {
+
+  const std::string contents = text;
+
+  chrome_lang_id::NNetLanguageIdentifier lang_id;
+  const chrome_lang_id::NNetLanguageIdentifier::Result lang_id_result =
+      lang_id.FindTopNMostFreqLangs(contents, /*num_langs=*/1).at(0);

```

Comment:
nit: in ads/b-c we usually use `lang_id.FindTopNMostFreqLangs(contents, /* num_langs */ 1).at(0);`